base/acme/webapps/acme/index.jsp

```diff
@@ -0,0 +1,139 @@
+<!--
+Copyright Red Hat, Inc.
+
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+<html>
+<head>
+    <title>ACME Responder</title>
+    <link rel="stylesheet" href="css/patternfly-4.35.2.css">
+    <script src="js/jquery-3.5.1.js"></script>
```

Comment:
Right now yes because ACME might be deployed separately from other subsystems, and it's a new UI so we want to use the latest libraries. I'd rather not change the other subsystems' UI libraries now since it requires a lot of manual testing. Hopefully in the future we can upgrade all libraries to the same level so we can combine them, but right now they need to be separate.